org.eclipse.scanning.points/src/org/eclipse/scanning/jython/JythonInterpreterManager.java

```diff
@@ -0,0 +1,243 @@
+
+            File lib = getBundleLocation(jythonBundleName); // TODO Name the jython OSGi bundle without Diamond in it!
+	    	Iterator<Object> it = state.path.iterator();
+	    	while (it.hasNext()) {
+	    		Object ob = it.next();
+	    		if (ob instanceof String && ((String)ob).endsWith(File.separator+"Lib")) {
+	    			lib = new File((String)ob);
+	    		}
+	    	}
+	    	
+	    	if (lib==null) {
+	            File loc = getBundleLocation(jythonBundleName); // TODO Name the jython OSGi bundle without Diamond in it!
+           	    File jythonDir = find(loc, "jython");
+            	lib       = find(jythonDir, "Lib");
+	    	}
+	    	
+            if (lib != null) {
+            	File site       = find(lib, "site-packages");
+            	state.path.add(new PyString(site.getAbsolutePath())); // Resolves the collections
+
+            	File[] fa = site.listFiles();
+            	for (File dir : fa) {
+            		if (!dir.isDirectory()) continue;
+            		if (dir.getName().endsWith("-info")) continue;
+            		state.path.add(new PyString(dir.getAbsolutePath())); // Resolves the collections
+            	}
+            	
+            	state.toString();
+
+            	System.out.println(state.path);
+            }
+            
+        } catch (Exception ne) {
+        	System.out.println("Problem setting jython path to include scripts");
+        	ne.printStackTrace();
+        	logger.debug("Problem setting jython path to include scripts!", ne);
+        }
+ 	}
+
+	private static ClassLoader getBundleLoader(String name) {
+    	Bundle bundle = Platform.getBundle(name);
+    	BundleWiring bundleWiring = bundle.adapt(BundleWiring.class);
+    	return bundleWiring.getClassLoader();
+	}
+	   
+    /**
+	 * @param bundleName
+	 * @return File
+	 */
+	private static File getBundleLocation(final String bundleName) {
+		
+		try {
+			final Bundle bundle = Platform.getBundle(bundleName);
+			if (bundle == null) {
+				throw new IOException();
+			}
+			return FileLocator.getBundleFile(bundle);
+		}
+		catch (IOException e) {
+			File dir = new File("../"+bundleName);
+			if (dir.exists()) return dir;
+			
+			dir = new File("../../diamond-jython.git/"+bundleName);
+			if (dir.exists()) return dir;
+			
+			dir = new File("../../scanning.git/"+bundleName);
+			if (dir.exists()) return dir;
+			
+			// These paths refer to finding things in the travis build 
+			// They will not resolve from the IDE or binary.
+			dir = new File("../../org.eclipse.scanning/"+bundleName);
+			if (dir.exists()) return dir;
+			
+			dir = new File("../../diamond-jython/"+bundleName);
+			if (dir.exists()) return dir;
+			
+			dir = new File("../../../diamond-jython/"+bundleName);
+			if (dir.exists()) return dir;
+		}
+		return null;
+	}
+    
+    private static File find(File loc, String name) {
+    	
+    	if (loc==null) return null;
+    	if (!loc.exists()) return null;
+    	File find = new File(loc, name);
+        if (find.exists()) return find;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)